### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by evanvosberg** [annexare/Countries#33]
With the latest update I run into `ReferenceError: window is not defined` error, when loading `countries-list` in node environment.

The error is caused in the module exports wrapper in this `./dist/index.es5.min.js`

```
!(function(n, a) {
  "object" == typeof exports && "object" == typeof module
    ? (module.exports = a())
    : "function" == typeof define && define.amd
      ? define("Countries", [], a)
      : "object" == typeof exports
        ? (exports.Countries = a())
        : (n.Countries = a());
})(window, function() {
```

Here window is used without checking whether it's defined.

**Comment by z-ax** [comment]
Hi. Thank you for reporting this, looks like a Webpack 4 issue. Checking.

**Comment by z-ax** [comment]
@evanvosberg fixed in v2.4.1, please verify.

**Comment by evanvosberg** [comment]
Thanks for this very quick fix. Just tested and it works again.
